codegen-server/src/main/kotlin/software/amazon/smithy/rust/codegen/server/smithy/protocols/RestJson1.kt

```diff
@@ -256,7 +239,7 @@ class RestJson1HttpSerializerGenerator(
         val responseTests =
             operationShape
                 .getTrait<HttpResponseTestsTrait>()
-                ?.getTestCasesFor(AppliesTo.SERVER)
+                ?.getTestCasesFor(AppliesTo.CLIENT)
```

Comment:
I think you want server? the client ones are client-only weird behavior

Reply:
I want server yes, and I forget about changing this back during my tests. Thanks for the catch. I agree, sooner or later we will want a separate class. I just made this work, so hopefully I will be able to start working on the real protocol tests already available :)